src/CommunityToolkit/Xamarin.CommunityToolkit/Views/Calendar/CalendarWeekDayHeader.shared.cs

```diff
@@ -0,0 +1,26 @@
+using System;
+using Xamarin.Forms;
+
+namespace Xamarin.CommunityToolkit.UI.Views
+{
+	/// <summary>
+	/// The <see cref="CalendarWeekDayHeader"/> represents specific week day in top header of <see cref="Calendar"/>.
+	/// </summary>
+	public class CalendarWeekDayHeader : TemplatedView
+	{
+		/// <summary>
+		/// Gets day of week.
+		/// </summary>
+		public DayOfWeek DayOfWeek { get; }
+        
+		/// <summary>
+		/// Initializes a new instance of the <see cref="CalendarWeekDayHeader"/> class.
+		/// </summary>
+		/// <param name="dayOfWeek">day of the week</param>
+		public CalendarWeekDayHeader(DayOfWeek dayOfWeek)
+		{
+			DayOfWeek = dayOfWeek;
+			var a = DayOfWeek; // Without this line ControlTemplate doesn't work.
```

Comment:
Can you talk more about it? I'm curious why that happens and what happened if we remove this line. Is it failing on XCT or in the original project as well?

Reply:
Without this line week day headers at the top of calendar won't show:
![Screenshot 2021-05-30 at 17 04 27](https://user-images.githubusercontent.com/12022141/120109328-3f244300-c169-11eb-920d-4e73d6f099b7.png)

This was not happening in the original project.

Reply:
> Without this line week day headers at the top of calendar won't show
Do you know why?

Reply:
no idea :(

Reply:
Could you investigate and resolve it?

I’d prefer not to merge this workaround into the Toolkit.

Reply:
me neither :) I will investigate this

Reply:
@brminnick I've investigated this and have some conclusions. If I delete line 23 from CalendarWeekDayHeader and set "Link behavior" to "Do not link" for "Debug | iPhoneSimulator " in Xamarin.CommunityToolkit.Sample.iOS, then week day headers are properly displayed at the top of calendar. It seems that linker is somehow responsible for this issue. 

What would you suggest?

Reply:
on CalendarView ctor, add this line
`_ = new CalendarWeekDayHeader (dumbDayOfWeek);`

and let me know if it works.

Reply:
@pictos I've deleted line 23 from CalendarWeekDayHeader and added suggested line by you to Calendar ctor - this didn't solve the issue. Week day headers at the top of calendar are still not visible

Reply:
@DamianAntonowicz I'll give some love on this issue during this week and let you know if I find any fix.

Reply:
@pictos did you had time to take a look at this?

Reply:
Fixed